tests/pass/shims/x86/intrinsics-x86-bmi.rs

```diff
@@ -0,0 +1,154 @@
+// Ignore everything except x86 and x86_64
+// Any new targets that are added to CI should be ignored here.
+// (We cannot use `cfg`-based tricks here since the `target-feature` flags below only work on x86.)
+//@ignore-target-aarch64
+//@ignore-target-arm
+//@ignore-target-avr
+//@ignore-target-s390x
+//@ignore-target-thumbv7em
+//@ignore-target-wasm32
+//@compile-flags: -C target-feature=+bmi1,+bmi2
+
+#[cfg(target_arch = "x86")]
+use std::arch::x86::*;
+#[cfg(target_arch = "x86_64")]
+use std::arch::x86_64::*;
+
+fn main() {
```

Comment:
Usually these tests have a `assert!(is_x86_feature_detected!("..."));` to make sure they are safe to copy-paste and run in rustc as well. Please add that here.

Reply:
Done.